### PR TITLE
add banner frontsite and backoffice

### PR DESCRIPTION
**Pull request by ntarocco**
Blocked by the release on Invenio-Banners.

## Frontsite

banner_info_home

## Backoffice

backoffice-banner

### CODE REVIEW

File: src/lib/components/Banner/Banner.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import _isEmpty from 'lodash/isEmpty';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Message } from 'semantic-ui-react';
+
+const FETCH_BANNER_EVERY_SECS = 60 * 5; // 5 minutes
+
+const BannerCmp = ({ message, category }) => {
+  var colorProp = {};
```
Comment (FlorianCassayre) [inline src/lib/components/Banner/Banner.js:9]: `let` instead?

Comment (ntarocco) [reply src/lib/components/Banner/Banner.js]: fixed

File: src/lib/components/Banner/Banner.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import _isEmpty from 'lodash/isEmpty';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Message } from 'semantic-ui-react';
+
+const FETCH_BANNER_EVERY_SECS = 60 * 5; // 5 minutes
```
Comment (kprzerwa) [inline src/lib/components/Banner/Banner.js:6]: no websockets ? ;)

File: src/lib/components/Banner/Banner.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      colorProp['warning'] = true;
+      break;
+    case 'info':
+      colorProp['info'] = true;
+      break;
+    default:
+      break;
+  }
+  return (
+    <Message className="center no-margin" {...colorProp}>
```
Comment (kprzerwa) [inline src/lib/components/Banner/Banner.js:21]: Do we want to provide overridable?

File: src/lib/routes/backoffice/BackOffice.js [hunk 4]
```diff
@@ -1,22 +1,25 @@
...
     const { location } = this.props;
     return (
-      <div className="backoffice">
-        <div className="bo-sidebar">
-          <Sidebar location={location} />
-        </div>
-        <div className="bo-content">
-          <Notifications />
-          <BackOfficeRoutesSwitch />
+      <Banner>
```
Comment (kprzerwa) [inline src/lib/routes/backoffice/BackOffice.js:12]: why on the backoffice the `<Banner>` has children and for the frontsite not?

**Comment by ntarocco** [comment]
Fixed all comments and also many console warning fixes.

**Review by kprzerwa — APPROVED** [review]

**Pull request merged**